code/firmware/rosco_m68k_firmware/stage2/main2.c

```diff
@@ -177,6 +197,7 @@ noreturn void lmain() {
 #  if (defined SDFAT_LOADER) || (defined IDE_LOADER)
     FW_PRINT_C("No bootable media found\r\n");
 #  endif
+start_uart_loader:
```

Comment:
Getting `main2.c:200:1: error: label 'start_uart_loader' defined but not used` for this when building without the menu (e.g. `ROMDEVICE=SST39SF010` or non-HUGEROM builds).

Reply:
I will add more ifdefs.

Reply:
added